docs/readmes/nms/alert.md

```diff
@@ -0,0 +1,322 @@
+|ue_reported_usage / up	|Reported TX traffic for subscriber / session in bytes	|AGW	|Yes	|sessiond	|
+|ue_reported_usage / down	|Reported RX traffic for subscriber / session in bytes	|AGW	|Yes	|sessiond	|
+|ue_dropped_usage / up	|Reported dropped TX traffic for subscriber / session in bytes	|AGW	|No	|sessiond	|
+|ue_dropped_usage / down	|Reported dropped RX traffic for subscriber / session in bytes	|AGW	|No	|sessiond	|
+
+More up to date information might be available from the “Metrics Explorer” in the metrics component.
+![alerts4](assets/nms/userguide/alerts/alerts4.png)
+**Custom Alert Configuration**
+
+An alert configuration consists of
+
+* Name/Description of the alert
+* Alert Definition
+* Alert receiver to be notified when the alert is fired
+* Additional labels which can be added to provide more information about the alert.
+
+Alert definition is the most interesting attribute in the configuration. It consists of a metric expression(based on prometheus promQL expression) and a duration attribute which specifies the time for the expression to be true, following which the alert is fired.
+![alerts5](assets/nms/userguide/alerts/alerts4.png)![alerts6](assets/nms/userguide/alerts/alerts4.png)
+
+Following examples shows how we can use the above mentioned metrics and add custom alerts based on them.
+
+**eNB Down Alert:**
+This alert fire if eNB Rf Tx is down in any of the gateways in your network.
+
+Expression:
+
+```
+sum by(gatewayID) (enodeb_rf_tx_enabled{networkID="<your network ID>"} < 1)
+```
+
+Duration:
+10 minutes
+
+**No Connected eNB Alert:**
+This alert fire if the connected eNB count falls to ‘0’ for any of the gateways in your network.
+
+Simple: Select “enb_connected” metric from the dropdown and construct the if statement as **“if enb_connected < 1”**
+
+Advanced Expression:
+
+```
+(enb_connected{networkID="<your network ID>"}) < 1
+```
+
+Duration:
+5 minutes
+
+**Free Memory is < 10 Alert:**
+This alert fire if the Free Memory of any of the gateways in your network is less than 10%
+
+Advanced Expression:
+
+```
+((1 - mem_available{networkID="<your network ID>"} / mem_total{networkID=~"<your network ID>"}) * 100) > 90
+```
+
+Duration:
+15 minutes
+
+**High Disk Usage Alert:**
+This alert fire if the disk usage of any of the gateways in your network is more than 80%
+Simple Way: Select “disk*_percent*” metric from the dropdown and construct the if statement as **“if disk*_percent* > 80”**
+Advanced Expression:
+
+```
+(disk_percent{networkID="<your network ID>"}) > 80
+```
+
+Duration:
+15 minutes
+
+**Attach Success Rate Alert:**
+This alert fire if the attach success rate of any of the gateways in your network is less than 50% for a 3h window.
+Expression:
+
+```
+(sum by(gatewayID) (increase(ue_attach{action="attach_accept_sent",networkID="<your network ID>"}[3h]))) * 100 / (sum by(gatewayID) (increase(ue_attach{action=~"attach_accept_sent|attach_reject_sent|attach_abort",networkID="<your network ID>"}[3h]))) < 50
```

Comment:
Since this is pretty complex you might want to break it down and explain what's going on. That might be out of scope here since it would just be explaining PromQL. Up to you if you think that would be useful.

Reply:
Added a brief explanation here.